### ISSUE TITLE
Add parameter to Get-DbaEstimatedCompletionTime to filter tasks that actually report progress

### ISSUE THREAD

**Issue by ben-thul** [dataplat/dbatools#8266]
### Summarize Functionality

Most commands in T-SQL don't populate percent_complete or estimated_completion_time in sys.dm_exec_requests. As written, Get-DbaEstimatedCompletionTime returns information for all sessions with an active request. On a busy server, that could be thousands of rows.

My proposal is to either:

* Change the default behavior of the cmdlet to exclude rows `WHERE percent_complete = 0 AND estimated_completion_time  = 0` and add a flag if the cmdlet caller wants the current behavior (i.e. "return information for all active sessions").
* Add a flag that allows the above behavior to be requested by the cmdlet caller.

### Is there a command that is similiar or close to what you are looking for?

No

### Technical Details

The internals of the cmdlet currently just run a query against sys.dm_exec_requests so inlining a WHERE clause based on the parameter in there should be fairly straightforward.

**Comment by andreasjordan** [comment]
I agree with the request and as the command name indicates, I would only return rows with a positive estimated_completion_time value.
What do you think @potatoqualitee ?

**Comment by potatoqualitee** [comment]
yes thank you

**Comment by andreasjordan** [comment]
Please have a look at my PR and give feedback...
